src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -45,7 +45,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
             return;
         }
-        if(userStore.hasAdmin(username)){
+        if(user.hasPrivileges() || userStore.hasAdmin(username)){
```

Comment:
What do you think about moving the username specific logic to the servlet that registers new users? That way the user object is the sole source of permission information and the only place "special username" handling is in the registration process when users are created.